config/routes.rb

```diff
@@ -15,6 +14,10 @@
 
   get "/", to: "application#redirection", as: :root_redirection
 
+  concern :paginatable do
+    get "(page/:page)", action: :index, on: :collection
+  end
```

Comment:
а це ще навіщо?)

Reply:
Because of the page parameter and Rails routing, you can easily generate SEO and user-friendly URLs.
This will create URLs like /products/page/33 instead of /products?page=33

it also has other added benefits...
Because the page parameter is now a URL segment, we can leverage on Rails page caching.